main/src/main/scala/sbt/Defaults.scala

```diff
@@ -156,7 +156,7 @@ object Defaults extends BuildCommon {
       javaHome :== None,
       testForkedParallel :== false,
       javaOptions :== Nil,
-      sbtPlugin :== false,
+      sbtPlugin := false,
```

Comment:
isn't this redundant now?

Reply:
I didn't want to accidentally change the semantics of `JvmPlugin` in case there's a direct use of `projectSettings`. In reality it's probably safe to remove it.

Reply:
I'd say let's remove it then.

Reply:
done